resources/assets/js/components/JobDetails/JobDetails.tsx

```diff
@@ -494,198 +567,118 @@ const JobDetails: React.FunctionComponent<
+                </p>
+                <p data-c-margin="bottom(normal)">
+                  <FormattedMessage
+                    id="jobDetails.remoteWorkGroupBody"
+                    defaultMessage="Want the best talent in Canada? You increase your chances when you allow those in other parts of Canada to apply. Regional diversity also adds perspective to your team culture. Make sure to discuss this in advance with your HR Advisor."
+                    description="Body message displayed on the remote work group input."
+                  />
+                </p>
+                <RadioGroupFormik
                   id="remoteWork"
                   label={intl.formatMessage(formMessages.remoteWorkGroupLabel)}
                   required
                   grid="base(1of1)"
                   error={errors.remoteWork}
                   touched={touched.remoteWork}
-                  info={
-                    <>
-                      <p
-                        data-c-margin="bottom(normal)"
-                        data-c-font-weight="bold"
-                      >
-                        <FormattedMessage
-                          id="jobDetails.remoteWorkGroupHeader"
-                          defaultMessage="Is remote work allowed?"
-                          description="Header message displayed on the remote work group input."
-                        />
-                      </p>
-                      <p data-c-margin="bottom(normal)">
-                        <FormattedMessage
-                          id="jobDetails.remoteWorkGroupBody"
-                          defaultMessage="Want the best talent in Canada? You increase your chances when you allow those in other parts of Canada to apply. Regional diversity also adds perspective to your team culture. Make sure to discuss this in advance with your HR Advisor."
-                          description="Body message displayed on the remote work group input."
-                        />
-                      </p>
-                    </>
-                  }
-                  value={values.remoteWork}
                 >
-                  <Field
-                    name="remoteWork"
-                    component={RadioInput}
-                    id="remoteWorkWorld"
-                    label={intl.formatMessage(
-                      formMessages.remoteWorkWorldLabel,
-                    )}
-                  />
-                  <Field
-                    name="remoteWork"
-                    component={RadioInput}
-                    id="remoteWorkCanada"
-                    label={intl.formatMessage(
-                      formMessages.remoteWorkCanadaLabel,
-                    )}
+                  {remoteWorkOptions.map(
+                    ({ id, value, label }): React.ReactElement => {
+                      return (
+                        <Field
+                          name="remoteWork"
+                          component={InputFormik}
```

Comment:
I'm not sure that the markup is similar enough to warrant generalizing a radio input into a default `Input` component here.